### PR TITLE
delete this

### PR DESCRIPTION
**Pull request by matmen**

### CODE REVIEW

**Review by ConnorMcF — APPROVED** [review]

**Review by NotSoSuper — CHANGES_REQUESTED** [review]
you forgot to delete your self

**Review by Kefta — APPROVED** [review]

**Review by GGG-KILLER — APPROVED** [review]
Very good commit, I agree

**Comment by KittoniuM** [comment]
I agree with you

File: LICENSE [hunk 1]
```diff
@@ -1,21 +0,0 @@
-MIT License
```
Comment (mcd1992) [inline LICENSE]: MIT LICENSE IS GARBAGE PLEASE SWITCH TO GPL THANKS

**Review by mcd1992 — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (ConnorMcF) [inline LICENSE]: GPL IS GARBAGE PLEASE SWITCH TO WTFPL THANKS

Comment (GGG-KILLER) [inline LICENSE]: WTFPL IS GARBAGE PLEASE SWITCH TO UNLICENSE THANKS

Comment (RaidAndFade) [reply LICENSE]: SOMETHING THAT DOESNT EXIST DOESNT NEED GPL BUDDY NO LICENSE REQUIRED FOR THIS TRASH FEEL FREE TO USE AND ABUSE

**Comment by RaidAndFade** [comment]
👍

**Review by RaidAndFade — APPROVED** [review]
:+1: Just do it already

File: LICENSE (hunk 1 above)
Comment (xmatmen) [reply LICENSE]: EXACTLY

**Review by Tvde1 — APPROVED** [review]

**Comment by Fogapod** [comment]
Nice changes! Want to see this in the next update. Please merge

**Review by Fogapod — APPROVED** [review]

File: LICENSE (hunk 1 above)
Comment (hatf0) [reply LICENSE]: UNLICENSE IS GARBAGE PLEASE SWITCH TO AGPL THANKS

Comment (rv32ima) [inline LICENSE:1]: UNLICENSE IS GARBAGE PLEASE SWITCH TO AGPL THANKS

**Comment by matmen** [comment]
@NotSoSuper okay i deleted myself; can this be merged now?

**Review by ozzt — APPROVED** [review]

**Comment by TheRealLegitCuck** [comment]
Everybody here sucks

**Comment by ConnorMcF** [comment]
You're here too.

**Comment by ThatLing** [comment]
Please fix the merge conflicts

**Comment by empt1xo** [comment]
hello

**Comment by KittoniuM** [comment]
DO NO DO THIS
I WILL NOT FORGIVE

> For he must not do this, and deletion of NotSoBot will remain false

#23

**Comment by matmen** [comment]
pls do

**Comment by Kefta** [comment]
@NotSoSuper give me repo admin and I'll merge this real quick

**Comment by matmen** [comment]
@NotSoSuper technically i am deleted (from Discord), are you gonna merge this now

**Comment by empt1xo** [comment]
this ^

**Review by Tvde1 — CHANGES_REQUESTED** [review]
fix conflicts, then merge

**Comment by matmen** [comment]
will do

**Comment by matmen** [comment]
closing this because #70 is more up to date and has no conflicts

**Comment by ConnorMcF** [comment]
re-opening as the license change is crucial.

**Comment by empt1xo** [comment]
please merge.

**Pull request closed without merging**